IFComp/lib/IFComp/Schema/Result/User.pm

```diff
@@ -399,6 +399,15 @@ sub send_password_reset_email {
     $self->_send_email_and_reset_token('reset_password');
 }
 
+sub send_judge_reminder_email {
```

Comment:
Can we change this method to `send_author_reminder_email`?

Reply:
Oh, and tidy please.

Reply:
Thanks - not sure what I was thinking. I also need to tidy, but I'll do that after work tonight